index.js

```diff
@@ -5,5 +5,9 @@ module.exports = function (buffer) {
 		return false;
 	}
 
-	return buffer[20] === 0x68 && buffer[21] === 0x65 && buffer[22] === 0x69 && buffer[23] === 0x63;
+	const [b1, b2, b3, b4] = Array.from(buffer.slice(20, 24));
```

Comment:
@coolassassin Can you please drop the `Array.from` and destructuring? I would like to save support of older nodejs versions. Also, can you please add the test case for this? Thanks!